modules/persistent-storage-flexvolume-installing.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-flexvolume.adoc
+
+[id="flexvolume-installing_{context}"]
+
+= Installing FlexVolume drivers
+
+FlexVolume drivers that are used to extend {product-title} are executed only on the node. To implement FlexVolumes, a list of operations to call and the installation path are all that is required.
+
+.Prerequisites
+
+* FlexVolume drivers must implement these operations:
+
+`init`::
+Initializes the driver. It is called during initialization of all nodes.
+
+** Arguments: none
+** Executed on: node
+** Expected output: default JSON
+
+`mount`::
+Mounts a volume to directory. This can include anything that is necessary
+to mount the volume, including attaching the volume to the node, finding the device, and then mounting the device.
+
+** Arguments: `<mount-dir>` `<json>`
+** Executed on: node
+** Expected output: default JSON
+
+`unmount`::
+Unmounts a volume from a directory. This can include anything that is
+necessary to clean up the volume after unmounting, such as detaching the volume
+from the node.
+
+** Arguments: `<mount-dir>`
+** Executed on: node
+** Expected output: default JSON
+
+* All other operations should return JSON with `{"status": "Not supported"}` and exit code `1`.
+
```

Comment:
Flexvolume's api documentation is kinda incomplete in Openshift. This documentation does not document `mountdevice` and `unmountdevice` api calls. In addition we do not document resizing support for flexvolume plugins. 

A more complete documentation is available on - https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md and https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/grow-flexvolume-size.md

Reply:
Thanks for the info and links. 

- I didn't include `mountdevice` and `unmountdevice` before because they were documented in "FlexVolume drivers with master-initiated attach/detach" section and I was not including attach/detach info. I've added those calls in the doc assembly. 

- I'm currently working on getting FlexVolume expansion content added (WIP).

Reply:
@gnufied I've finished adding FlexVol content from the upstream links you shared. Tried to place content where it made most sense (e.g., FlexVol expansion under the "Expanding PVs" section). 
PTAL - thanks!